receiver/sshcheckreceiver/testdata/config.yaml

```diff
@@ -3,7 +3,7 @@ receivers:
     endpoint: notdefault:1313
     username: notdefault_password
     password: notdefault_password
-    keyfile: notdefault/path/keyfile
+    key_file: notdefault/path/keyfile
```

Comment:
Is this fix related?

Reply:
I needed to use this test data to test the change and noted that it was outdated. Should I update it in a different PR?

Reply:
Maybe best. Interesting that you can change this and no tests failed. Making it a separate pr or issue would force us to look at validating that file going forward.

Reply:
Created #24739 for this